qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -53,21 +60,53 @@ class ResilienceOptions:
             Only applicable for ``resilience_level=2``.
             Default: "TwoQubitAmplifier".
 
-        extrapolator: An extrapolation strategy. One of ``"LinearExtrapolator"``,
+        extrapolator (DEPRECATED): An extrapolation strategy. One of ``"LinearExtrapolator"``,
             ``"QuadraticExtrapolator"``, ``"CubicExtrapolator"``, ``"QuarticExtrapolator"``.
             Note that ``"CubicExtrapolator"`` and ``"QuarticExtrapolator"`` require more
             noise factors than the default.
             Only applicable for ``resilience_level=2``.
             Default: "LinearExtrapolator".
+
+        measure_noise_mitigation: Whether to enable measurement error mitigation method.
+            By default, this is enabled for resilience level 1, 2, and 3 (when applicable).
+
+        zne_mitigation: Whether to turn on Zero Noise Extrapolation error mitigation method.
+            By default, ZNE is enabled for resilience level 2.
+        zne_noise_factors: An list of real valued noise factors that determine by what amount the
+            circuits' noise is amplified.
+            Only applicable if ZNE is enabled.
+            Default: (1, 3, 5).
+        zne_extrapolator: An extrapolation strategy. One or more of ``"multi_exponential"``,
+            ``"single_exponential"``, ``"double_exponential"``, ``"linear"``.
+            Only applicable if ZNE is enabled.
+            Default: "linear" and "multi_exponential"
+
+        pec_mitigation: Whether to turn on Probabilistic Error Cancellation error mitigation method.
+            By default, PEC is enabled for resilience level 3.
+
     """
 
     noise_amplifier: NoiseAmplifierType = None
-    noise_factors: Sequence[float] = (1, 3, 5)
-    extrapolator: ExtrapolatorType = "LinearExtrapolator"
+    noise_factors: Sequence[float] = None
+    extrapolator: ExtrapolatorType = None
+
+    # Measurement error mitigation
+    measure_noise_mitigation: Optional[bool] = None
+
+    # ZNE
+    zne_mitigation: Optional[bool] = None
+    zne_noise_factors: Sequence[float] = (1, 3, 5)
+    zne_extrapolator: Optional[Union[ZneExtrapolatorType, Sequence[ZneExtrapolatorType]]] = field(
+        default_factory=lambda: ["linear", "multi_exponential"]
```

Comment:
Can you avoid default factor if you use a tuple? Either way the default value should be `("exponetial", "linear")` in that order
```suggestion
        default_factory=lambda: ("exponential", "linear")
```